Assets/Editor/BrushLister.cs

```diff
@@ -36,7 +36,9 @@ static void ListBrushes()
         foreach (BrushDescriptor brush in experimentalBrushes)
         {
             try { AppendValidBrushString(brush, true); }
-            catch (Exception UnassignedReferenceException) { }
+            catch (Exception UnassignedReferenceException) { 
```

Comment:
This was throwing an unused variable warning and logging these exceptions will probably help debugging in the future.